conf/common/test/nemo_run_llama3_8b_lora.toml

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+# Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Let's use 2025 for new files (this and others as well).

P.S. I'll prioritize the fix for copyright test.